### PR TITLE
docs(v2:) Divide `markdown-features/code-blocks` to smaller sections

### PR DESCRIPTION
**Pull request by ramiy**
This PR adds few more heading to split the content to small topics.

This way users like me will be able to scroll directly to the specific section, instead of reading all the documentation.

I add to read the entire section because It was not divided into sections. Not a great user experience.

### CODE REVIEW

**Comment by ramiy** [comment]
Code-blocks-Docusaurus

File: website/docs/guides/markdown-features/markdown-features-code-blocks.mdx [hunk 1]
```diff
@@ -142,6 +146,8 @@ html[data-theme='dark'] .docusaurus-highlight-code-line {
 }
 ```
 
+## Multiple line highlighting {#multiple-line-highlighting}
```
Comment (Josh-Cena) [inline website/docs/guides/markdown-features/markdown-features-code-blocks.mdx:149]: I think the structure should be more like:

```md
## Line highlighting
### Single line highlighting
### Multiple line highlighting
### Highlighting with comments
```

Also, be ware that the section titles should be in sentence case.

Comment (ramiy) [reply website/docs/guides/markdown-features/markdown-features-code-blocks.mdx]: Good catch, thank you for noticing. It was fixed.

**Comment by slorber** [comment]
Thanks 👍

**Pull request merged**